src/test/ui/consts/control-flow/assert.rs

```diff
@@ -1,14 +1,14 @@
 // Test that `assert` works when `const_panic` is enabled.
 
-// revisions: stock panic
+// revisions: stock const_panic
 
-#![cfg_attr(panic, feature(const_panic))]
+#![cfg_attr(const_panic, feature(const_panic))]
```

Comment:
From adding the feature gate I did notice that this new `cfg` predicate can of course now clash with user-set `cfg`.

I'm not sure if that's problematic enough to prevent stabilization, or perhaps requires that we can only stabilize with an edition?